GothicVRProject/GothicVR
# Feature/smooth walking and lab npc

Merging after https://github.com/GothicVRProject/GothicVR/pull/279 only!

# To test

GVR:
1. Feature-Flags - Activate NPCs, NPC routines, and spawn only NPC=233 (Bloodwyn)
2. Check if he's walking on the ground

Lab:
1. Load Lab and disable XRMockController (if active)
2. Walk to Bloodwyn
3. Select an animation
4. Play animation
